### PR TITLE
Scenario notebook

### PR DESCRIPTION
**Pull request by nickmalleson**
This PR integrates the changes that @spoonerf made to the model for the paper. The main addition is the notebook: https://github.com/Urban-Analytics/RAMP-UA/blob/scenario_notebook/experiments/calibration/week_earlier_2021.ipynb as well as a relatively minor addition to allow a healthier population. (I'm not sure what this does, but is off by default)

Once this has been merged we can tag the repo as it represents the state of the model used to create the results.

### CODE REVIEW

**Review by Sparrow0hawk — COMMENTED** [review]
Added a couple of quick questions on this. I want to have a closer play with the notebook and will look at creating some amendments to get it working via binderhub if possible.

**Comment by Sparrow0hawk** [comment]
Couldn't get the experiment notebook working in BinderHub (due to memory limits) but will have a look at providing some configuration for doing it in colab. Happy to approve this once minor comments above are resolved.

File: experiments/opencl_runner.py [hunk 1]
```diff
@@ -312,11 +338,14 @@ def run_opencl_model_multi(
...
         l_store_detailed_counts = [store_detailed_counts] * repetitions
-        l_quiet = [True] * repetitions  # Don't print info
+        l_quiet = [False] * repetitions  # Don't print info
+
 
-        args = zip(l_i, l_iterations, l_snapshot_filepath, l_params, l_opencl_dir, l_use_gpu,
-                   l_store_detailed_counts, l_quiet)
+        args = zip(l_i, l_iterations, l_snapshot_filepath, l_params, l_opencl_dir, l_use_gpu, 
+                   l_store_detailed_counts, l_quiet)            #print(params[0])
+        args = zip(l_i, l_iterations, l_snapshot_filepath, l_params, l_opencl_dir, l_use_gpu, l_use_healthier_pop, l_store_detailed_counts, l_quiet)
```
[the remark this answers is not in the document]
Comment (nickmalleson) [reply experiments/opencl_runner.py:348]: Agreed. Looks like I messed up a merge

File: gam_cases_new.csv [hunk 2]
```diff
@@ -0,0 +1,110 @@
+"","x"
```
[the remark this answers is not in the document]
Comment (nickmalleson) [reply gam_cases_new.csv:1]: Yeah agreed, the whole case files organisation is a bit of a mess. They're duplicated as well I think. I have created an issue to do this properly: https://github.com/Urban-Analytics/RAMP-UA/issues/295

**Comment by nickmalleson** [comment]
@Sparrow0hawk thanks for looking at this, I think I have resolved all conversations unless I have missed soemthing. If you're happy could you merge?

**Review by Sparrow0hawk — APPROVED** [review]

**Pull request merged**